python/chemiscope/input.py

```diff
@@ -72,6 +72,35 @@ def create_input(frames, meta=None, properties=None, cutoff=None):
     will generate four properties named ``cheese[1]``, ``cheese[2]``,
     ``cheese[3]``,  and ``cheese[4]``, each containing 300 values.
 
+    Here is a quick example of generating a chemiscope input from an `ase.Atoms`_
+    file, a `json`_-type metadatafile, and a text file which contains additional
+    atomic properties as columns, with headers.
+
+    .. code-block:: python
+
+       from ase.io import read
+       import numpy as np
+       import json
+
+       input_file = 'trajectory.xyz'
+       property_file = 'properties.txt'
+       meta_file = 'metadata.json'
+
+       frames = read(input_file, ':')
+       property_values = np.genfromtxt(property_file, names=True)
```

Comment:
I'm not sold on using the very specific `np.genfromtxt` function in an example, and loading data from files in general. 

For this example, we could have an ellispis around how data is loaded, something like 
```py
# example 1: list containing the energy of each structure, from a set of calculations
energy = [ ... ]

# example 2: PCA projection computed using sklearn [LINK].
# X contains a multi-dimensional descriptor of the structure
X = np.array( ... )
pca = sklearn.decomposition.PCA(n_components=3).fit_transform(X)

properties = {
    "PCA": {
        "target": "atom",
        "values": pca,
    },
    "energies": {
        "target": "structure"
        "values": energies,
        "units": "kcal/mol", # maybe leave this one out of the example?
    },
}
```

This also removes the use of a dictionary comprehension, which can be confusing to beginners.